docs/timelines.md

```diff
@@ -0,0 +1,133 @@
+- You can mute tracks to prevent them playing back using the \{Power\} button.
+- You can lock tracks to prevent accidental changes to sections you've finished using the \{Lock\} button.
+- Tracks can be collapsed so they take up less room on the screen, using the arrow button to the left of the track name.
+
+Selecting timecode source
+------------------
+
+You can select one of the four available timecode sources to control your timeline from the [Timeline Options](./timelines/timeline-options.md#timecode-tab).
+
+To set up and enable each of the timecode sources as Internal, MIDI, Clock or SMPTE use the \[Timecode\] softkey at the top level main menu.
+
+Local timecode controls
+------------------
+
+The \{Link\} button in the bottom left of the Timeline View window links or unlinks the timeline with the timecode source. Especially with external timecode, it's useful when recording and editing to be able to control the timecode yourself rather than having to continually ask the video guys to rewind the source. So when this button is set to unlinked, you can control the timeline playback using the buttons in the top left corner of the Timeline View window as you would with an audio player. If you are using Internal Timecode as a timecode source then these buttons will also operate the Internal Timecode source while linked.
+
+![Timeline transport controls](/docs/images/Timeline-Timecode-Transport.png)
+
+- \{ |< \} sets the live timecode back to zero.
+- \{ |> \} starts playing from the start timecode.
+- \{ > \} and \{ || \} are play / pause controls.
+- \{ Stop \} stops playing and sets the live timecode back to zero.
+- \{ Record \} enters live record mode but doesn't start timecode playback.
+
+When you are linked to a timecode (except Internal Timecode), the first four buttons are replaced by the timecode source name and only the Record button is shown.
+
+![Timeline external timecode controls](/docs/images/Timeline-Timecode-ExternalTransport.png)
+
+If the timecode source is disabled in the **Timecode** menu, the time is shown in red.
+
+![Timeline disabled](/docs/images/Timeline-Timecode-Disabled.png)
+
+- If your timecode source starts at a big number, you can set a time offset to allow you to use zero-referenced times on your timeline.
+See [Time Offset](./timelines/timeline-options.md#time-options).
+
+The overview bar
+------------------
+
+The overview bar at the bottom of the screen gives you a view of the whole timeline from start to end. Triggers and markers are shown in miniature on the bar. You can make the main view zoom in and out by dragging the end markers of the bar to enclose a particular section. It's a good way of moving quickly around the timeline.
+
+![Timeline overview bar](/docs/images/Timeline-Overview-Bar-Cropped.png)
+
+Moving around the timeline view with the wheels
+-----------------
+
+Using the context menu option \{Timeline Wheels\} you can set the wheels to control your view.
```

Comment:
Might be worth a mention that this is only the case if you don't have any triggers selected